golang/modules/lambda_module_context.go

```diff
@@ -0,0 +1,33 @@
+package modules
+
+import (
+	"context"
+	"github.com/kurtosis-tech/kurtosis-client/golang/core_api_bindings"
+	"github.com/palantir/stacktrace"
+)
+
+type ModuleID string
+
+// Docs available at https://docs.kurtosistech.com/kurtosis-libs/lib-documentation
+type LambdaModuleContext struct {
+	client core_api_bindings.ApiContainerServiceClient
+	moduleId ModuleID
+}
+
+func NewLambdaModuleContext(client core_api_bindings.ApiContainerServiceClient, moduleId ModuleID) *LambdaModuleContext {
+	return &LambdaModuleContext{client: client, moduleId: moduleId}
+}
+
+// Docs available at https://docs.kurtosistech.com/kurtosis-libs/lib-documentation
+func (moduleCtx *LambdaModuleContext) Execute(argsJsonStr string) (responseJsonStr string, resultErr error) {
+	args := &core_api_bindings.ExecuteLambdaArgs{
+		ModuleId:   string(moduleCtx.moduleId),
+		ParamsJson: argsJsonStr,
+	}
+	resp, err := moduleCtx.client.ExecuteLambda(context.Background(), args)
```

Comment:
Here you can use responseJsonStr and resultErr instead of declaring new variables